samples/openapi3/client/petstore/python-experimental/test/test_drawing.py

```diff
@@ -41,130 +34,12 @@ def setUp(self):
     def tearDown(self):
         pass
 
-    def test_create_instances(self):
```

Comment:
So some of the files in the tests directory have real tests.
We see that this method test_create_instances has a manually written test in it.
This update is deleting those real tests.
Please preserve them?

Reply:
Let me take a stab at cleaning this up as well.

I want to open a seperate core issue around this in general.  The fact that we both commit the model tests and don't regenerate model tests when present masked this issue for me.  (This is neither a huge priority, nor is there a single obvious fix for it)

Reply:
Can I suggest a fix? This has come up a couple of times.
How about:
- Moving these manually written tests into the tests folder
- Renaming the tests folder something like tests_manual

Reply:
I pulled out all but one of the sample test changes.  Seeing that we have both the generated test stubs and real tests, I'm not sure how much value there is in keeping all of the stubs up to date.

Reply:
Missed that last round of suggestions - let me take a run at that.

Reply:
Moved `tests` to `tests_manual`.

Moved the following from `test` into `tests_manual`:
> samples/openapi3/client/petstore/python-experimental/test/test_drawing.py
> samples/openapi3/client/petstore/python-experimental/test/test_fake_api.py
> samples/openapi3/client/petstore/python-experimental/test/test_shape.py
> samples/openapi3/client/petstore/python-experimental/test/test_fruit_req.py
> samples/openapi3/client/petstore/python-experimental/test/test_gm_fruit.py
> samples/openapi3/client/petstore/python-experimental/test/test_fruit.py

Regenerated all tests in the `test` directory.

Squashed everything down to one commit - I hated to do it, but the reason some of the tests were generating incorrectly was drift from master since I started working on this more than a week back.

Reply:
Thank you very much!